apps/web/src/app/(dashboard)/templates/[id]/page.tsx

```diff
@@ -1,81 +1,12 @@
 import React from 'react';
 
-import Link from 'next/link';
-import { redirect } from 'next/navigation';
+import type { TemplatePageViewProps } from './template-page-view';
+import TemplatePageView from './template-page-view';
```

Comment:
Purposely did default import since the `PageView` should be considered the page itself.

No go?

Reply:
Since it's a View (which is basically a page) I reckon we could make it a little more nice in terms of props api, since realistically if we're updating the params or searchParams we'd receive it wouldn't be super seamless to begin with anyway.